Playlist_Genre_Country.php

```diff
@@ -88,51 +95,51 @@
 									ORDER BY Song_ID ASC");
 
 
-						//Runs and stores the query using the variable $con (see nav.php) and $query
-						$result = mysqli_query($conn,$query);
-						//runs in a 'while' loop
-						while($output=mysqli_fetch_array($result))
-						{
-						?>
-						<!--php is above. HTML is below. Used to output the query results -->
+							//Runs and stores the query using the variable $con (see nav.php) and $query
+							$result = mysqli_query($conn,$query);
+							//runs in a 'while' loop
+							while($output=mysqli_fetch_array($result))
+							{
+							?>
+							<!--php is above. HTML is below. Used to output the query results -->
+
 
+							<heading2>
+								<space></space>
 
-					<heading2>
-						<space></space>
+								<Song_ID2><p2 class = 'white'><?php echo $output['Song_ID']; ?></p2></Song_ID2>
+								<img src="images/<?php echo $output['Images']; ?>" style="width: 50px; height: 50px;">
+								<Title2><p2 class = 'white'><?php echo $output['Title']; ?></p2></Title2>
+								<Artist2><p2 class = 'white'><?php echo $output['Artist']; ?></p2></Artist2>
+								<Album2><p2 class = 'white'><?php echo $output['Album']; ?></p2></Album2>
+								<Genre2><p2 class = 'white'><?php echo $output['Genre']; ?></p2></Genre2>
+								<Size2><p2 class = 'white'><?php echo $output['Size']; ?></p2></Size2>
+								<Duration2><p2 class = 'white'><?php echo $output['Seconds']; ?></p2></Duration2>
+								<space></space>
 
-						<Song_ID2><p class = 'white'><?php echo $output['Song_ID']; ?></p></Song_ID2>
+							</heading2>
+							<?php
+								//Closes the output while loop
+							}
+							?>
 
-						<img src="images/<?php echo $output['Images']; ?>" style="width: 50px; height: 50px;">
 
+						</content>
 
-						<Title2><p class = 'white'><?php echo $output['Title']; ?></p></Title2>
-						<Artist2><p class = 'white'><?php echo $output['Artist']; ?></p></Artist2>
-						<Album2><p class = 'white'><?php echo $output['Album']; ?></p></Album2>
-						<Genre2><p class = 'white'><?php echo $output['Genre']; ?></p></Genre2>
-						<Size2><p class = 'white'><?php echo $output['Size']; ?></p></Size2>
-						<Duration2><p class = 'white'><?php echo $output['Seconds']; ?></p></Duration2>
-						<space></space>
 
-					</heading2>
-					<?php
-					//Closes the output while loop
-					}
-					?>
+					</div>
+				</div>
 
-				</content>
 			</div>
 		</div>
-	
-	
-		<br>
+
 		<br>
 		<br>
-	
-	
 		<!-- Footer -->
-		<div class="w3-bottom footer w3-center w3-black w3-padding-32">
-			<p1>Created by <a target="_blank" class="w3-hover-text-orange">Soundwave</a> &copy; Copyright Liv Amer 2022</p1>
-		</div>
+		<footer class="w3-bottom w3-center w3-grey w3-padding-16">
```

Comment:
added a bottom code as well as made the navbar grey, also changed the padding to 16 rather than 64 (was too big)